edit.php

```diff
@@ -0,0 +1,86 @@
+<?php
+$db = new PDO("mysql:host=192.168.20.20;dbname=Portfolio", 'root', '');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+if (!empty($_GET['id'])) {
+    $sql = 'SELECT `id`, `title`, `img_url`, `site_url` FROM `projects` WHERE `id`= ?;';
+    $query = $db->prepare($sql);
+    $query->execute([$_GET['id']]);
+    $result = $query->fetch();
+} else {
+    echo 'ERROR: $_GET[id] is empty';
+}
+
+if(!isset($_POST['cancelBtn'])) {
```

Comment:
can we split this code into a seperate file? rather than making the edit form submit to itself